hack/addons/update-calico-manifests.sh

```diff
@@ -13,17 +13,15 @@ if [ -z "${CALICO_VERSION:-}" ]; then
   exit 1
 fi
 
-CALICO_CNI_ASSETS_DIR="$(mktemp -d -p "${TMPDIR:-/tmp}")"
-readonly CALICO_CNI_ASSETS_DIR
-trap 'rm -rf ${CALICO_CNI_ASSETS_DIR}' EXIT
+ASSETS_DIR="$(mktemp -d -p "${TMPDIR:-/tmp}")"
+readonly ASSETS_DIR
+trap 'rm -rf ${ASSETS_DIR}' EXIT
```

Comment:
nit: to be consistent with NFD
```suggestion
trap_add "rm -rf ${ASSETS_DIR}" EXIT
```